platform-emf/src/main/java/com/softicar/platform/emf/table/listener/EmfTableListenerSettings.java

```diff
@@ -0,0 +1,29 @@
+package com.softicar.platform.emf.table.listener;
+
+import com.softicar.platform.db.core.connection.DbConnections;
+
+public class EmfTableListenerSettings {
```

Comment:
- I think this deserves some Javadoc since it relies on the rather unusual principle of interacting with the current transaction (which is not obvious at all).
- I think we should have a test for this.

Reply:
added javadoc and test